exercises/bob/Cargo.lock

```diff
@@ -1,4 +0,0 @@
-[root]
```

Comment:
I think we should delete all Cargo.lock files, I don't see any advantage keeping them around.